Source/WebKit/UIProcess/API/Cocoa/_WKSystemPreferences.mm

```diff
@@ -39,26 +39,25 @@ @implementation _WKSystemPreferences
 
 + (BOOL)isCaptivePortalModeEnabled
 {
-    auto key = adoptCF(CFStringCreateWithCString(kCFAllocatorDefault, WKLockdownModeEnabledKey.characters(), kCFStringEncodingUTF8));
+    auto key = adoptCF(CFStringCreateWithCStringNoCopy(kCFAllocatorDefault, WKLockdownModeEnabledKey.characters(), kCFStringEncodingUTF8, kCFAllocatorNull));
```

Comment:
Is it documented that `CFPreferencesCopyValue` is guaranteed to never retain the passed in key? Or are we just guessing based on common sense?

Reply:
I mean characters is a const global variable of a C++ string.

Should not matter honestly because retain counts don't affect this.